edith-step/src/App.js

```diff
@@ -1,15 +1,31 @@
 import React, {Component} from 'react';
-import ReceiptInput from './ReceiptInput';
-// import "./App.css";
 
+import ReceiptInput from './components/ReceiptInput';
+import TopNavbar from './components/TopNavbar';
 
-/** Main webpage for the website. */
+import './App.css';
+
+/**
+ * Main webpage for the website.
+ */
```

Comment:
nit: I think we can make this an inline javadoc

`/** Main webpage for the website. */`

Reply:
I was inline before but I got a lint error and changed it.